src/libraries/System.Security.Cryptography.Algorithms/tests/DefaultECDsaProvider.Unix.cs

```diff
@@ -21,18 +21,9 @@ public bool IsCurveValid(Oid oid)
             return IsValueOrFriendlyNameValid(oid.FriendlyName);
         }
 
-        public bool ExplicitCurvesSupported
-        {
-            get
-            {
-                if (PlatformDetection.IsOSXLike)
-                {
-                    return false;
-                }
+        public bool ExplicitCurvesSupported => true;
```

Comment:
Why are you changing the truth value for macOS?

Reply:
This was a typo. I think I accidentally thought I was in the .Android file. I'll fix this.

Reply:
This was due to a bad rebase.

Reply:
Fixed by doing a rebase to a more recent commit on main.